### ISSUE TITLE
Detour creating too many false positives with slow networks

### ISSUE THREAD

**Issue by myleshorton** [getlantern/lantern#2803]

**Comment by myleshorton** [comment]
Based on what we've seen from users in Iran, connection times of 10 seconds or more are not uncommon. As a result, detour will have a lot of false positives because it uses a threshold of connecting in 3 seconds or less for determining if a site is blocked. Does that all sound correct @fffw?

**Comment by fffw** [comment]
Yes it is what detour currently do. Detour v2 which uses parallel access
still has lots of bugs but I expect a 2-3 days effort to fix them.

-- 
BR,
Merlin
